app/views/group_offers/show.html.slim

```diff
@@ -65,7 +65,7 @@ h1= @group_offer.title
             td= t_attr(:schedule_details)
             td= @group_offer.schedule_details
 
-- if @group_offer.responsible
+- if @group_offer.responsible && !@group_offer.external?
```

Comment:
the other way arround because its cheaper: !offer.external? && offer.responsible

Or better make a method in the model